src/com/xilinx/rapidwright/design/tools/ArrayBuilder.java

```diff
@@ -612,19 +747,43 @@ public static void main(String[] args) {
             }
             modules.add(m);
         }
-        t.stop().start("Place Instances");
 
         Design array = null;
         List<String> modInstNames = null;
+        List<Pair<Pair<Integer, Integer>, String>> idealPlacementList = null;
```

Comment:
I think longer term, we may want to create an object here, but for now, just add a comment to describe what all the fields stand for.  A `List<Pair<Pair<Integer, Integer>, String>>` is a bit abstract.